aiohttp/payload.py

```diff
@@ -107,27 +126,27 @@ def __init__(self, value, *, headers=None, content_type=sentinel,
         self._content_type = content_type
 
     @property
-    def size(self):
+    def size(self) -> Optional[float]:
```

Comment:
This seems a little awkward, but honestly speaking that's how I read the logic. Might as well be that I got that wrong.